setup.py

```diff
@@ -103,6 +103,7 @@ def run(self):
     'isort==5.1.4',
     'six>=1.13.0',
     'pydocstyle>=3.0.0',
+    'pytest-xdist>=1.33.0',
```

Comment:
We shouldn't add versions that are open to new versions. dependabot takes care of that and it prevents tests suddenly failing in the future without a change on our side.